de.metas.business/src/main/java/org/adempiere/impexp/product/CreateProductPriceCommand.java

```diff
@@ -0,0 +1,123 @@
+/**
+ *
+ */
+package org.adempiere.impexp.product;
+
+import static org.adempiere.model.InterfaceWrapperHelper.newInstance;
+import static org.adempiere.model.InterfaceWrapperHelper.save;
+
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+
+import org.adempiere.pricing.api.IPriceListDAO;
+import org.adempiere.util.Services;
+import org.compiere.model.I_C_TaxCategory;
+import org.compiere.model.I_M_PriceList;
+import org.compiere.model.I_M_PriceList_Version;
+import org.compiere.model.I_M_ProductPrice;
+
+import de.metas.pricing.ProductPrices;
+import lombok.NonNull;
+
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+/**
+ * @author metas-dev <dev@metasfresh.com>
+ *
+ */
+public class CreateProductPriceCommand
```

Comment:
why do you call it command?
i renamed it to ProductPriceImporter...pls ping me if you want to discuss..